system-int-test/test/cmr/system_int_test/ingest/deleted_granules_index_test.clj

```diff
@@ -53,3 +71,110 @@
+        granule3-prov1-2 (d/item->concept
+                          (dg/granule-with-umm-spec-collection
+                           collection-prov1-2 (:concept-id collection-prov1-2) {:concept-id "G3-PROV1"}))
+        granule4-prov2 (d/item->concept
+                        (assoc (dg/granule-with-umm-spec-collection
+                                 collection-prov2 (:concept-id collection-prov2) {:concept-id "G4-PROV2"})
+                               :provider-id "PROV2"))]
+    (ingest/ingest-concept granule1-prov1)
+    (ingest/delete-concept granule1-prov1)
+    (ingest/ingest-concept granule2-prov1)
+    (ingest/delete-concept granule2-prov1)
+    (ingest/ingest-concept granule3-prov1-2)
+    (ingest/delete-concept granule3-prov1-2)
+    (ingest/ingest-concept granule4-prov2)
+    (ingest/delete-concept granule4-prov2)
+    (index/wait-until-indexed)
+
+    (testing "Search for all"
+      (are3 [params granules]
+        (is (= (set (map :concept-id granules))
+               (set (find-deleted-granules params))))
+
+        "after tomorrow"
+        {:revision-date (t/plus- (t/now) (t/days 1))}
+        []
+
+        "after yesterday"
+        {:revision-date (t/minus- (t/now) (t/days 1))}
+        [granule1-prov1 granule2-prov1 granule3-prov1-2 granule4-prov2]))
+
+    (testing "Search for all after date for provider"
+      (are3 [params granules]
+        (is (= (set (map :concept-id granules))
+               (set (find-deleted-granules params))))
+
+        "PROV1"
+        {:revision-date (t/minus- (t/now) (t/days 1)) :provider "PROV1"}
+        [granule1-prov1 granule2-prov1 granule3-prov1-2]
+
+        "PROV2"
+        {:revision-date (t/minus- (t/now) (t/days 1)) :provider "PROV2"}
+        [granule4-prov2]))
+
+    (testing "Search for all after date for parent-collection-id"
+      (are3 [params granules]
+        (is (= (set (map :concept-id granules))
+               (set (find-deleted-granules params))))
+
+        "in PROV1, parent collection 1"
+        {:revision-date (t/minus- (t/now) (t/days 1))
+         :parent-collection-id (:concept-id collection-prov1)}
+        [granule1-prov1 granule2-prov1]
+
+        "in PROV1, parent collection 2"
+        {:revision-date (t/minus- (t/now) (t/days 1))
+         :parent-collection-id (:concept-id collection-prov1-2)}
+        [granule3-prov1-2]
+
+        "in PROV2, parent collection 1"
+        {:revision-date (t/minus- (t/now) (t/days 1))
+         :parent-collection-id (:concept-id collection-prov2)}
+        [granule4-prov2]))))
+
+(deftest deleted-granule-parameter-validation
+  (are3 [errors params]
+    (let [response (search/get-search-failure-data
+                    (search/find-deleted-granules params))]
+      (is (= (:status response) 400))
+      (is (= errors (:errors response))))
+
+    "Revision date range validation"
+    '("Revision date must be withing one year of today.")
```

Comment:
I don't think you need to use literal lists. `["Revision date must be withing one year of today."]` should work or just have a single error string passed into each test and change line 159 to `(is (= [error] (:errors response))))`